docs/book/src/cronjob-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -159,14 +168,17 @@ var _ = Describe("Manager", Ordered, func() {
 				"--", "/bin/sh", "-c", fmt.Sprintf(
 					"curl -v -k -H 'Authorization: Bearer %s' https://%s.%s.svc.cluster.local:8443/metrics",
 					token, metricsServiceName, namespace))
-			Expect(utils.Run(cmd)).Error().NotTo(HaveOccurred(), "Failed to create curl-metrics pod")
+			_, err = utils.Run(cmd)
+			Expect(err).Error().NotTo(HaveOccurred(), "Failed to create curl-metrics pod")
```

Comment:
.Error() here is not needed.
```suggestion
			Expect(err).NotTo(HaveOccurred(), "Failed to create curl-metrics pod")
```

Reply:
That is fine but I think is out of the scope of the PR right?
I think that is not related with the issue/motivation